homeassistant/components/zwave_js/sensor.py

```diff
@@ -235,6 +239,50 @@
         device_class=SensorDeviceClass.TEMPERATURE,
         native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
     ),
+    (
+        ENTITY_DESC_KEY_ENERGY_PRODUCTION_TIME,
+        UnitOfTime.SECONDS,
+    ): SensorEntityDescription(
+        ENTITY_DESC_KEY_ENERGY_PRODUCTION_TIME,
+        name="Energy production time",
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+    ),
+    (ENTITY_DESC_KEY_ENERGY_PRODUCTION_TIME, UnitOfTime.HOURS): SensorEntityDescription(
+        ENTITY_DESC_KEY_ENERGY_PRODUCTION_TIME,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.HOURS,
+    ),
+    (
+        ENTITY_DESC_KEY_ENERGY_PRODUCTION_TODAY,
+        UnitOfEnergy.WATT_HOUR,
+    ): SensorEntityDescription(
+        ENTITY_DESC_KEY_ENERGY_PRODUCTION_TODAY,
+        name="Energy production today",
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
I think we need to set `last_reset` on the entity if this measurement resets every day.

The third bullet in the examples:
https://developers.home-assistant.io/docs/core/entity/sensor#how-to-choose-state_class-and-last_reset